src/component/Invoice/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from "react"
+import moment from "moment"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronRight } from "@fortawesome/free-solid-svg-icons"
+
+function Invoice({ invoiceTotal, invoiceStatus, invoiceStatusName, clientName, invoiceId, dueDate }) {
+
+    const date = moment(dueDate);
+    const formattedDate = date.format("DD MMM YYYY");
+    const element = <FontAwesomeIcon icon={faChevronRight} />
+
+
+    useEffect(setColor, [])
+
+    const [statusColor, setStatusColor] = useState('')
+    const [statusBorderColor, setStatusBorderColor] = useState('')
```

Comment:
Best practice to put state at the top of a component - so you can find all of a components state straight away